### PR TITLE
Add /caddyshack/carl-spackler.md

### PR DESCRIPTION
**Pull request by JakeC360**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/1558)

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: english/staypuft/ghostbusters/billMurrayMovies/caddyshack/carl-spackler.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+As you near a state of total consciousness, your concentration is broken by
```
Comment (casey-collab) [inline english/staypuft/ghostbusters/billMurrayMovies/caddyshack/carl-spackler.md]: It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by JakeC360** [comment]
Please review

**Comment by cbuckey-uda** [comment]
@casey-collab

**Comment by casey-collab** [comment]
Looks great! :heavy_check_mark:

**Pull request merged**